### DISCUSSION TITLE
leerresultaten

### DISCUSSION THREAD

**Discussion by KoopReynders** [fdnd/docs.fdnd.nl#108] [Reflectie]
Werkwijze met issues, pull requests met de niveau matrix herschijven zonder de Github werkwijze? Lijkt me moeilijk te begrijpen voor een lezer die dit niet kent.

**Comment by ju5tu5** [comment]
Ik twijfel, het demonstreert namelijk ook heel mooi de iteratieve manier van werken. Driekwart van het panel kan hier prima mee uit de voeten..

**Comment by ju5tu5** [comment]
Opgenomen in de lopende tekst
